cpp/ps0004/Task4.cpp

```diff
@@ -41,3 +44,4 @@ int main()
 

```

Comment:
- [x] used `if` structure in a right way
- [x] `if` structure take in consideration all invalid cases
- [x] all expressions to check invalidity are correct
- [x] used `std::cout` and `std::endl` or `\n` to print a different message for the 3 cases
- [x] converted the number to the opposite printable character
- [x] printed the number and the opposite printable character